src/main/java/com/faforever/client/leaderboard/SubDivisionTabController.java

```diff
@@ -65,6 +67,10 @@ public void initialize() {
     gamesPlayedColumn.setCellValueFactory(param -> param.getValue().gamesPlayedProperty());
     gamesPlayedColumn.setCellFactory(param -> new StringCell<>(count -> i18n.number(count.intValue())));
 
+    winRateColumn.setCellValueFactory(param -> Bindings.createFloatBinding(() ->
+        Math.min((float) param.getValue().getScore() / param.getValue().getGamesPlayed() * 100, 100)));
+    winRateColumn.setCellFactory(param -> new StringCell<>(winRate -> i18n.rounded(winRate.doubleValue(), 1)));
```

Comment:
As BlackYPS said the score does not equal the number of games won so this is not accurate